### PR TITLE
WIP: Apply app theme to launcher widget (fixes #727)

### LINKED ISSUE #727: Apply app theme to launcher widget
IMG_20191008_044002

### PR DESCRIPTION
**Pull request by stefanfris**
Solves #727 

image

Changed 3 files to handle all cases i was able to notice. 
Dark mode now seem to work fine in widget.
Currently trying to solve issue when im able to apply text color only when i click on widget.
Please let me know if it is OK for now, it's my first issue.
Thanks :)

### CODE REVIEW

**Comment by gsantner** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/markor/markor/pullRequest?prid=4525321&bid=15238433)

**Comment by gsantner** [comment]
@stefanfris 
Thanks much for you help!

As far I can see theres same code on multiple places. Do you think it's possible to create a static method at one of the WrWidget classes, accepting e.g. Context & RemoteViews, and calling that one at all places?

**Comment by stefanfris** [comment]
@gsantner 
Sure, sounds good. I should have done it earlier.

File: app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java [hunk 1]
```diff
@@ -9,6 +9,8 @@
 #########################################################*/
 package net.gsantner.markor.activity;
 
+import android.appwidget.AppWidgetManager;
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java]: Codacy Issue found: [Unused import - android.appwidget.AppWidgetManager.](https://app.codacy.com/app/gsantner/markor/pullRequest?prid=4525321)

**Comment by gsantner** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/markor/markor/pullRequest?prid=4525321&bid=15238433)

File: app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java [hunk 2]
```diff
@@ -9,6 +9,8 @@
...
+import android.content.ComponentName;
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java]: Codacy Issue found: [Unused import - android.content.ComponentName.](https://app.codacy.com/app/gsantner/markor/pullRequest?prid=4525321)

Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java]: Codacy Issue found: [Avoid unused imports such as 'android.content.ComponentName'](https://app.codacy.com/app/gsantner/markor/pullRequest?prid=4525321)

File: app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java (hunk 1 above)
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/activity/SettingsActivity.java]: Codacy Issue found: [Avoid unused imports such as 'android.appwidget.AppWidgetManager'](https://app.codacy.com/app/gsantner/markor/pullRequest?prid=4525321)

**Comment by gsantner** [comment]
@stefanfris 
I see you made changes, nice :D . Does it work all as expected still? (both light/dark)

**Comment by gsantner** [comment]
@stefanfris 
 ?

**Comment by stefanfris** [comment]
@gsantner 

Oh, sorry I forgot to reply. Yes, it works still as described in first comment. I made a method as you recommended. But I wasn't able to solve issue I mentioned. Currently I'm trying to finish my semester so I'm little bit busy.

**Comment by gsantner** [comment]
@stefanfris 
Thanks for working on it. I'm merging it, we can improve when really needed

**Comment by stefanfris** [comment]
@gsantner 
Thank you too.

**Pull request merged**